lib/mvgApi.js

```diff
@@ -15,36 +15,36 @@ const iconv = require("iconv");
  */
 function getDepartures(station, options) {
```

Comment:
Change the signature to 
`function getDepartures(station, options, apiUrl = defaultApi)`